tests/drivers/lpuart/testcase.yaml

```diff
@@ -0,0 +1,42 @@
+tests:
+  lpuart.loopback_busy_sim:
+    platform_allow: nrf9160dk_nrf9160
```

Comment:
Please add `integration_platforms` entry to each test scenarios defined in this `yaml` file, to indicate explicitly what should be built/performed in CI.

Reply:
done